pkg/framework/operators/supported.go

```diff
@@ -14,5 +15,11 @@ var (
 			keepCdrs:     true,
 			apiVersion:   "v1alpha1",
 		}},
+		OperatorTypeBroker: &BrokerOperatorBuilder {BaseOperatorBuilder{
+			image: "brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/amq7/amq-broker-operator:0.9",
+			operatorName: "activemq-artemis-operator",
+			keepCdrs: true,
+			apiVersion: "v1alpha1",
+		}},
```

Comment:
Are we going to replace this with an upstream image? The idea is that it is not tied to any downstream stuff.

Reply:
Ouch, I've missed this.